test/integration/patients/worklist/worklist.js

```diff
@@ -2702,6 +2703,204 @@ context('worklist page', function() {
+          type: 'patients',
+          attributes: {
+            first_name: 'Test',
+            last_name: 'Patient',
+          },
+        });
+
+        return fx;
+      })
+      .routeSettings(fx => {
+        fx.data[0].attributes = {
+          value: {
+            widgets: ['sex', 'optionsWidget1'],
+            fields: [testField],
+          },
+        };
+
+        return fx;
+      })
+      .routeWidgets(fx => {
+        const addWidget = _.partial(getResource, _, 'widgets');
+
+        fx.data = fx.data.concat([
+          addWidget({
+            id: 'optionsWidget1',
+            widget_type: 'optionsWidget',
+            definition: {
+              display_name: 'Test Field Widget',
+              field_name: 'test-field',
+              display_options: { '1': 'Test Field' },
+            },
+          }),
+        ]);
+
+        return fx;
+      })
+      .routePatientField(fx => {
+        fx.data = getResource(testField, 'patient-fields');
+
+        return fx;
+      })
+      .routeFlow()
+      .routeFlowActions()
+      .visit('/worklist/owned-by')
+      .wait('@routeWidgets')
+      .wait('@routeFlows');
+
+    cy
+      .get('.app-frame__content')
+      .find('.table-list__item')
+      .first()
+      .as('firstRow');
+
+    cy
+      .get('@firstRow')
+      .find('.worklist-list__patient-sidebar-icon .js-patient-sidebar-button')
+      .click();
+
+    cy
+      .get('.app-frame__sidebar .sidebar')
+      .as('patientSidebar')
+      .find('.worklist-patient-sidebar__patient-name')
+      .should('contain', 'Test Patient');
+
+    cy
+      .get('@patientSidebar')
+      .find('.worklist-patient-sidebar__patient-info .js-patient')
+      .click();
+
+    cy
+      .url()
+      .should('contain', 'patient/dashboard/1');
+
+    cy
+      .get('.patient-sidebar')
+      .find('.patient-sidebar__name')
+      .should('contain', 'Test Patient');
+
+    cy
+      .go('back');
+
+    cy
+      .get('@firstRow')
+      .find('.worklist-list__patient-sidebar-icon .js-patient-sidebar-button')
+      .click();
+
+    cy
+      .get('@patientSidebar')
+      .find('.patient-sidebar__section')
+      .first()
+      .should('contain', 'Sex')
+      .should('contain', 'Female')
+      .next()
+      .should('contain', 'Test Field Widget')
+      .should('contain', 'Test Field');
+
+    cy
+      .get('@patientSidebar')
+      .find('.worklist-patient-sidebar__close-icon button')
+      .click();
+
+    cy
+      .get('@patientSidebar')
+      .should('not.exist');
+
+    cy
+      .get('.worklist-list__toggle')
+      .contains('Actions')
+      .click()
+      .wait('@routeActions');
+
+    cy
+      .get('.app-frame__content')
+      .find('.table-list__item')
+      .find('.worklist-list__patient-sidebar-icon .js-patient-sidebar-button')
+      .click();
+
+    cy
+      .get('@patientSidebar')
+      .find('.worklist-patient-sidebar__patient-name')
+      .should('contain', 'Test Patient');
+  });
+
```

Comment:
A quick breakdown for what I did in this test:

1. Open the worklist patient sidebar for a flow item.
2. Test the contents of the patient sidebar:
    - Verify the patient's name is shown.
    - Verify the `View Patient Dashboard` link goes to the correct `/patient/dashboard/:id` page. And to verify the page is loaded, get the patient's name from the patient sidebar on that page. Go back to the worklist page when done.
    - Verify that widgets are loaded/displayed. One basic `Sex` widget is shown. And another basic field widget is also shown.
    - Close the sidebar using the `X` icon.
3. Switch to the actions version of the worklist.
    - Open the sidebar and verify the sidebar is loading data by checking that the patient name is there. The other parts of the sidebar were already tested.